modules/installation-dns-user-infra.adoc

```diff
@@ -67,9 +67,9 @@ following records:
 [options="nowrap"]
 ----
 # _service._proto.name.                            TTL    class SRV priority weight port target.
-_etcd-server-ssl._tcp.<cluster_name>.<base_domain>  86400 IN    SRV 0        10     2380 etcd-0.<cluster_name>.<base_domain>.
-_etcd-server-ssl._tcp.<cluster_name>.<base_domain>  86400 IN    SRV 0        10     2380 etcd-1.<cluster_name>.<base_domain>.
-_etcd-server-ssl._tcp.<cluster_name>.<base_domain>  86400 IN    SRV 0        10     2380 etcd-2.<cluster_name>.<base_domain>.
+_etcd-server-ssl._tcp.<cluster_name>.<base_domain>.  86400 IN    SRV 0        10     2380 etcd-0.<cluster_name>.<base_domain>.
```

Comment:
I do not think we need the `.` at the end of `etcd-0.<cluster_name>.<base_domain>.`
@lihongan ^^

Reply:
@zhaozhanqi, does vSphere have some kind of DNS service? Where are you creating the DNS record?

I want to be sure that the issue isn't one of different DNS services expecting different input; Maybe one requires the terminal period, and another rejects it or accepts input both with and without the terminal period.

Reply:
we are using AWS route53 for vsphere cluster testing @jiajliu ^^ could you help confirm if I'm correct for above DNS records.  thanks